frontend/src/services/AdvisorService.ts

```diff
@@ -1,30 +1,38 @@
+import { getAuthToken } from "../utils/auth-helpers";
+
 /** Service function object to find all students given a search query
  * @param searchQuery  the search query
  * @param pageNumber  page number for the query
  * @param userToken the JWT token of the user
  */
-export const getStudents = (
-  searchQuery: string,
-  pageNumber: number,
-  userToken: string
-) =>
+export const getStudents = (searchQuery: string, pageNumber: number) =>
   fetch(`/api/users/students?search=${searchQuery}&page=${pageNumber}`, {
     method: "GET",
     headers: {
-      Authorization: "Token " + userToken,
+      Authorization: "Token " + getAuthToken(),
```

Comment:
are there any drawbacks to just doing this that I'm missing? Figured we don't need to use getAuthToken every file we call an endpoint

Reply:
Don't think so. We might as well refactor to just do this in every API call (in the future)